migrator.go

```diff
@@ -52,11 +53,12 @@ func (c Column) Nullable() (nullable bool, ok bool) {
 }
 
 func (c Column) DecimalSize() (precision int64, scale int64, ok bool) {
-	ok = c.precision.Valid && c.scale.Valid
-	if ok {
+	if ok = c.precision.Valid && c.scale.Valid; ok {
```

Comment:
seems need to change it to `ok = c.precision.Valid || c.scale.Valid` to get the correct precision info for `double`

Reply:
right! thank you!